lib/main_common.pm

```diff
@@ -273,7 +273,8 @@ sub is_repo_replacement_required {
       && !get_var('KEEP_ONLINE_REPOS')    # Set variable no to replace variables
       && get_var('SUSEMIRROR')            # Skip if required variable is not set (leap live tests)
       && !get_var('ZYPPER_ADD_REPOS')     # Skip if manual repos are specified
-      && !get_var('OFFLINE_SUT');         # Do not run if SUT is offine
+      && !get_var('OFFLINE_SUT')          # Do not run if SUT is offine
+      && !get_var('ZDUP');                # Do not run on ZDUP as this tests handle repos on their own
```

Comment:
s/this/these/ (it's either singular, this test, or plural, these tests)

Reply:
@asdil12 please fix this and we can merge